common-arcana.lic

```diff
@@ -40,6 +40,14 @@ module DRCA
     end
   end
 
+  def start_khris(khris, settings)
```

Comment:
With changes to `common-arcana`, we're going to see errors for people with multiple characters. Would you be up for putting just these changes in a separate PR?

Reply:
This shouldn't affect anyone as the methods are only called by people using buff. Barbs will be affected, but they just need to restart. So you're asking me to rename the activate_barb_buff back to start_barb_buff and then make a separate PR for the name change? Not sure how that's different from this?

Reply:
I forgot that people would have to opt-in to this by changing up their `buff` set. That's fine then.